res/themes/light/css/_light.scss

```diff
@@ -23,6 +23,7 @@ $secondary-fg-color: #737D8C;
 $tertiary-fg-color: #8D99A5;
 $header-panel-bg-color: #f3f8fd;
 
+$notification-badge-dot-color: darkblue;
```

Comment:
You have changed the default element behaviour, which we want to avoid.  Instead you will need to set this style to be the same value as $primary-fg-color and move the setting of this new style to dark blue, into the same file in aether-host-files

Reply:
Hi Jasen,  $primary-fg-color is dark-brownish-black and the requirement was explicit to make it dark blue? Could you please confirm if i still need to make it  $primary-fg-color. It may fail in the test phase.

Reply:
Samridhi I wasn't changing the requirement, just the implementation, so the _light.scss file in this repo needs:
`$notification-badge-dot-color: #2e2f32;`
then in the _light.scss file in the aether-host-files repository, it can have:
`$notification-badge-dot-color: darkblue;`

Reply:
Thanks Jasen, I am done with this change.